environment

```diff
@@ -1,5 +1,5 @@
 export PATH=%MINC_TOOLKIT_DIR%/bin:/data/%PROJECT%/bin/mri:/data/%PROJECT%/bin/mri/uploadNeuroDB:/data/%PROJECT%/bin/mri/uploadNeuroDB/bin:/data/%PROJECT%/bin/mri/dicom-archive:$PATH
-export PERL5LIB=/data/%PROJECT%/bin/mri/uploadNeuroDB:$PERL5LIB
+export PERL5LIB=/data/%PROJECT%/bin/mri/uploadNeuroDB:/data/%PROJECT%/bin/mri/dicom-archive:$PERL5LIB
```

Comment:
@davidblader this line is important so that the scripts can find the library in dicom-archive. I will add a note for existing project that they need to update their environment file so that it looks like:
```
export PERL5LIB=/data/%PROJECT%/bin/mri/uploadNeuroDB:/data/%PROJECT%/bin/mri/dicom-archive:$PERL5LIB
```

Reply:
right, i did that. this will works for accessing libraries, but the issue is when trying to run a different script using `$FindBin::Bin` which returns the directory of the script currently executing. 
for example, when running dcmconvTarchive.pl, it returns an error because it is looking for `tools/dicomTar.pl`